File: 2538/2538.cpp
class Solution {
public:
    int differenceBetweenMaximumAndMin(TreeNode* root) {
        if(!root) return 0;
        return 1+max(differenceBetweenMaximumAndMin(root->left),differenceBetweenMaximumAndMin(root->right));
    }
};

File: 2538/2538.cs
public class Solution {
    public int DifferenceBetweenMaximumAndMin(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(DifferenceBetweenMaximumAndMin(root.left), DifferenceBetweenMaximumAndMin(root.right));
    }
}